gravitee-am-ui/src/app/components/sidenav/sidenav.component.html

```diff
@@ -34,7 +34,10 @@
       </a>
     </gio-menu-list>
     <gio-menu-footer *ngIf="canDisplayOrganizationSettings()">
-      <a *ngFor="let item of footerMenuItems" [routerLink]="item.path" title="{{item.label}}">
+      <a *ngFor="let item of footerMenuItems"
+         [routerLink]="item.path"
+         title="{{item.label}}"
+         [matTooltip]="item.tooltip" [matTooltipPosition]="'right'">
```

Comment:
Is it for accessibilty reason that the title is defined to the label value ?
Due to title definition and the matTooltip definition we have two labels displayed with two different value.

Reply:
after a sync with UI/UX team, title has to be removed.